src/main/java/in/twizmwaz/cardinal/module/modules/kit/KitCollection.java

```diff
@@ -0,0 +1,101 @@
+package in.twizmwaz.cardinal.module.modules.kit;
+
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.event.CycleCompleteEvent;
+import in.twizmwaz.cardinal.module.modules.filter.FilterModule;
+import in.twizmwaz.cardinal.module.modules.filter.FilterState;
+import org.bukkit.Material;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KitCollection implements KitRemovable {
```

Comment:
If you look at https://docs.oc.tc/modules/kits, the 3 values that i kept as booleans in the parent kits are there because they will always be set to what you set them to, or to the default, the idea of making kit subclases, is so that it adds the posibility to make one do stuff, but don't have the other one do anything

A perfect example was the walk speed kit, in the old system, walk speed was set to 0.2F by default, and changed to something else if specified, well, the kit didn't work on boomBox / ender blast because it was first applying the modified kit, and afterwards (parent kit) applied default speed again